pallets/msa/src/rpc/src/lib.rs

```diff
@@ -78,14 +78,12 @@ where
 		let provider = Provider(provider_msa_id);
 
 		Ok(delegator_msa_ids
-			.par_iter()
+			.iter() // TODO: Change back to par_iter()
```

Comment:
par_iter?

Reply:
That's a future TODO.  We are intentionally reverting it to .iter().  @wilwade  will be making a more permanent fix which uses par_iter() again.